app/controllers/ContactHmrcController.scala

```diff
@@ -104,11 +105,12 @@ class ContactHmrcController @Inject() (
             val submit = routes.ContactHmrcController.submit(service, userAction, referrerUrl)
             Future.successful(BadRequest(contactHmrcPage(formWithErrors, submit)))
           },
-          data =>
+          (data: ContactForm) =>
             (for {
               maybeEnrolments <- enrolmentsConnector.maybeAuthenticatedUserEnrolments()
               _               <- createDeskproTicket(data, maybeEnrolments)
             } yield {
+              logger.warn(s"Contact form submitted: $data")
```

Comment:
@kyle-bowden we need to remove this, it could leak PII

Reply:
Left over from POC, removed it now.